tests/net/npf/src/main.c

```diff
@@ -72,6 +75,34 @@ static struct net_pkt *build_test_pkt(int type, int size, struct net_if *iface)
 	return pkt;
 }
 
+static struct net_pkt *build_test_ip_pkt(void *src, void *dst,
+	   sa_family_t family, struct net_if *iface)
+{
+	struct net_pkt *pkt;
+	int ret = -1;
+	int size;
+
+	size = (family == AF_INET) ? sizeof(struct net_ipv4_hdr) :
+		(family == AF_INET6) ? sizeof(struct net_ipv6_hdr) : 0U;
+
+	pkt = net_pkt_rx_alloc_with_buffer(iface, size, family, 0, K_NO_WAIT);
+	zassert_not_null(pkt, "");
+
+	if (family == AF_INET) {
+		ret = net_ipv4_create(pkt, (struct in_addr *)src,
+						(struct in_addr *)dst);
```

Comment:
Some weird indentation here - generally we try to align with the opening bracket when we split the argument list.

Reply:
Now should be better.